src/task-cycler.test.js

```
@@ -0,0 +1,244 @@
+      ];
+      
+      // Clear the output array to ensure we have a clean state
+      consoleOutput = [];
+      
+      await testHandleBreakSession();
+      
+      // Debug logs to help understand what's happening
+      console.error('Debug - consoleOutput:', consoleOutput);
+      
+      assert.strictEqual(consoleOutput[0], 'Break over. Time to work!');
+      assert(consoleOutput[1] && consoleOutput[1].includes('Your next task:'));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)